rules/endpoint/elastic_endpoint.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2020/07/08"
+ecs_version = ["1.5.0"]
+maturity = "development"
```

Comment:
we'll need to bump these to `production` to make it into the package

Reply:
```suggestion
maturity = "production"
```